internal/blockchain/fabric/fabric.go

```diff
@@ -567,3 +567,23 @@ func parseContractLocation(location fftypes.Byteable) (*Location, error) {
 	}
 	return fabricLocation, nil
 }
+
+func (f *Fabric) ValidateFFI(ctx context.Context, ffi *fftypes.FFI) error {
```

Comment:
Not sure we need this one in each plugin. Might only need the more granular methods below.